packages/site/src/layout/SearchBox.tsx

```diff
@@ -56,18 +57,26 @@ export const SearchBox = ({
   };
 
   return (
-    <Modal open={open} onRequestClose={closeModal} title="Search">
-      <Content>
+    <Modal size="large" open={open} onRequestClose={closeModal} title="Search">
+      <Content spacing={"larger"}>
         <InputText
           ref={ref}
           value={search}
           placeholder="Search"
+          prefix={{ icon: "search" }}
+          clearable="always"
           onChange={d => setSearch(d as string)}
         />
-        <Box width="100%">
```

Comment:
See **[comment](https://github.com/GetJobber/atlantis/pull/2100#issuecomment-2453119640)** on why Box didn't meet the needs here